database/redis/notification.go

```diff
@@ -98,7 +130,119 @@ func (connector *DbConnector) removeNotifications(notifications []*moira.Schedul
 }
 
 // FetchNotifications fetch notifications by given timestamp and delete it
-func (connector *DbConnector) FetchNotifications(to int64) ([]*moira.ScheduledNotification, error) {
+func (connector *DbConnector) FetchNotifications(to int64, limit int64) ([]*moira.ScheduledNotification, error) {
+	// No limit
+	if limit == 0 {
```

Comment:
I would like to see number as a constant `notificationsLimitUnlimited` or something like this

Reply:
Done